build.gradle

```diff
@@ -48,6 +48,15 @@ subprojects {
     }
   }
 
+  javadoc {
+    source = sourceSets.main.allJava
+    classpath = configurations.compileClasspath
+    options {
+      setMemberLevel JavadocMemberLevel.PUBLIC
```

Comment:
This is how you control what is rendered to javadocs. You can emit package protected or private elements. In this case, we only want to publish the public because that is what is usable. Really doesn't trim out that many classes.